config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_secretmanagersecretversions.secretmanager.cnrm.cloud.google.com.yaml

```diff
@@ -75,23 +78,11 @@ spec:
                   to be base64-encoded string and would be sent as is.
                 type: boolean
               resourceID:
-                description: Immutable. Optional. The service-generated name of the
-                  resource. Used for acquisition only. Leave unset to create a new
-                  resource.
+                description: Immutable. The SecretVersion name. If not given, the
+                  metadata.name will be used.
                 type: string
               secretData:
                 description: Immutable. The secret data. Must be no larger than 64KiB.
-                oneOf:
```

Comment:
If we wanted to keep this, we could add logic here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/9eeeae7b620288ab5854bda51ebd0cb19aa7f4b8/scripts/add-validation-to-crds/parse-crds.go#L142